apps/tsconfig.build.json

```diff
@@ -32,5 +32,6 @@
     "resolveJsonModule": true,
     "incremental": true,
     "sourceMap": true,
+    "downlevelIteration": true,
```

Comment:
this allows us to do `for...of` type iteration in typescript ([reference](https://www.typescriptlang.org/tsconfig#downlevelIteration))